apps/src/templates/studioHomepages/RecentCourses.jsx

```diff
@@ -30,15 +41,26 @@ const RecentCourses = React.createClass({
           showLink={showAllCoursesLink}
           isRtl={isRtl}
         >
+          {!isTeacher && studentTopCourse && (
+            <StudentTopCourse
+              isRtl={isRtl}
+              assignableName={studentTopCourse.assignableName}
+              lessonName={studentTopCourse.lessonName}
+              linkToOverview={studentTopCourse.linkToOverview}
+              linkToLesson={studentTopCourse.linkToLesson}
+            />
+          )}
           {topFourCourses.length > 0 && (
             topFourCourses.map((course, index) =>
+            <div key={index}>
               <CourseCard
-                key={index}
                 name={course.name}
                 description={course.description}
                 link={course.link}
                 isRtl={isRtl}
               />
+              {(index % 2 === 0) && <div style={styles.spacer}>.</div>}
```

Comment:
For what it's worth, [here's a work-in-progress diff](https://github.com/code-dot-org/code-dot-org/pull/16335/files) of my login-type cards using flexbox layout.

![image](https://user-images.githubusercontent.com/1615761/28041739-72724026-657f-11e7-9ffe-0d50a35ccff7.png)

They can flow with content height, which is why my second row of cards (without age recommendations) is shorter here - but we could still easily do fixed-height cards just by specifying a height, too.

I haven't tested thoroughly on IE11 yet but this shouldn't hit any of [the known pitfalls](http://caniuse.com/#feat=flexbox) with `flex-direction: column` and `min-height`.